requirements/pip.txt

```diff
@@ -64,13 +65,14 @@ commonmark==0.5.5
 recommonmark==0.4.0
 
 # Version comparison stuff
-Distutils2==1.0a3
 packaging==16.8
 
 # Commenting stuff
-git+https://github.com/zestedesavoir/django-cors-middleware.git@def9dc2f5e81eed53831bcda79ad1c127ecb3fe2
+django-cors-middleware==1.3.1
```

Comment:
I'm not sure why this one was pinned to a commit. @ericholscher any background here?